examples/basics/data_rows.ipynb

```diff
@@ -2,18 +2,22 @@
   "cells": [
```

Comment:
<p>Could you change "develop" to "master". We are currently setting all notebooks to point to master since the links are static and won't get updated when we merge from develop to master. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>It looks like it is master already. Are you seeing in other links?</p>

Reply:
<p>nevermind someone went through and already updated all of this.</p>